unlock-app/src/__tests__/components/interface/Error.test.js

```diff
@@ -36,8 +41,20 @@ describe('Error Component', () => {
   describe('the the component has an error message', () => {
     it('should display the content of the children', () => {
       const message = <p>There was an error.</p>
-      const wrapper = rtl.render(<Error close={close} error={message} />)
+      const wrapper = rtl.render(<Errors close={close} errors={[message]} />)
       expect(wrapper.queryByText('There was an error.')).not.toBeNull()
     })
   })
+
+  describe('connecting to redux', () => {
+    it('mapStateToProps', () => {
+      let state = []
+      state = errorReducer(state, setError('error 1'))
+      state = errorReducer(state, setError('error 2'))
+      state = errorReducer(state, setError('error 3'))
+      expect(mapStateToProps({ errors: state })).toEqual({
```

Comment:
Of course this is possible, but I have been burned in the past by tests that passed and should have failed because the underlying data they received was incorrect. If the structure of the result returned from errorReducer were to change, the test would continue to pass, but mSP would be returning incorrect data. So, what I was trying to test here was the interface between mSP and the reducer. I'll remove it if you still object, as this is testing integration, not unit.

Reply:
Then we need an integration for this rather than here ;)